SmartDeviceLink/SDLFocusableItemLocator.m

```diff
@@ -127,14 +132,15 @@ - (nullable UIView *)viewForPoint:(CGPoint)point {
     for (UIView *view in self.focusableViews) {
         //Convert the absolute location to local location and check if that falls within view boundary
         CGPoint localPoint = [view convertPoint:point fromView:self.viewController.view];
+        localPoint.x /= self.scale;
```

Comment:
I think the `point` has already been scaled by the time it gets here so it doesn't need to be scaled again.

Reply:
fixed